.github/workflows/release.yml

```diff
@@ -0,0 +1,127 @@
+name: release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+
+permissions: {}
+
+jobs:
+  build:
+    if: (github.repository == 'rubocop/rubocop')
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    outputs:
+      artifact-id: ${{ steps.upload-artifact.outputs.artifact-id }}
+      gem_name: ${{ steps.build.outputs.gem_name }}
+      gem_version: ${{ steps.build.outputs.gem_version }}
+      gem_path: ${{ steps.build.outputs.gem_path }}
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - uses: ruby/setup-ruby@d8d83c3960843afb664e821fed6be52f37da5267 # pin@v1.231.0
+        with:
+          ruby-version: ruby # Latest stable CRuby version
+          bundler-cache: false # dot not using caching in the SLSA build process
+
+      # IMPORTANT: this step MUST export for the following outputs:
+      # gem_name: the name of the gem - ex: "rubocop"
+      # gem_version: the version of the gem - ex: "1.0.0"
+      # gem_path: the path/filename of the gem - ex: "rubocop-1.0.0.gem"
+      - name: build
+        id: build
+        run: script/build
```

Comment:
Hm, any reason to not just inline this? I think the stuff with the colors and such is not really necessary. The next step should also show in logs what values it used from the env.

Reply:
We could absolutely in-line it. Totally up to the preference of the maintainers I suppose.